lua/cfc_block_luarun/config.lua

```diff
@@ -0,0 +1,32 @@
+local ALLOWED_LUA = {
+	"f3a9e5157f25d221d2b3387f55cc22c6", -- gm_excess_island_night
```

Comment:
it might be better to be a bit more specific eg:
```lua
-- gm_excess_island_night
"377582bb92686b70a4a7377b1de6f3a3", -- does thing
"7da014290063534942cfb9ad31fae513", -- adds wire support for map
```

Reply:
Would it be alright if I just replaced the comments with the actual code? (commented)

Reply:
Might be difficult if there's a lot of code, but mentioning the map that it's from is definitely a must

Reply:
I agree about mentioning the map, I made a comment before the items with the map name. Here is a screenshot of what I've done. What do you think?
![Code_yegyKlfUfx](https://user-images.githubusercontent.com/82342827/192167779-30a157db-61a6-456d-8b40-750f451272c2.png)

Reply:
What's like the longest lua string in a lua_run entity you found? and they don't have indentation right?

Reply:
They're just one long string of code. As for the longest one, there are three that are the same but are longer than the others.
I haven't committed anything yet, but lines 6, 8, and 9 are the lines in my file:

if Wire_CreateInputs then local controlbox=ents.FindByName('wiremain_b11d')[1];local door=ents.FindByName('b11_celldoor3')[1];controlbox.LightEdge=false;controlbox.Inputs=Wire_CreateInputs(controlbox,{'Door 3 - Control','Door 3 - Toggle'});function controlbox:TriggerInput(iname,value) if(iname=='Door 3 - Control') then if(value>0) then door:Fire('Open','',0) else door:Fire('Close','',0) end else if (value>0) and not self.LightEdge then door:Fire('Toggle','',0) end self.LightEdge=value>0 end end end

if Wire_CreateInputs then local controlbox=ents.FindByName('wiremain_b11b')[1];local door=ents.FindByName('b11_celldoor2')[1];controlbox.LightEdge=false;controlbox.Inputs=Wire_CreateInputs(controlbox,{'Door 2 - Control','Door 2 - Toggle'});function controlbox:TriggerInput(iname,value) if(iname=='Door 2 - Control') then if(value>0) then door:Fire('Open','',0) else door:Fire('Close','',0) end else if (value>0) and not self.LightEdge then door:Fire('Toggle','',0) end self.LightEdge=value>0 end end end

if Wire_CreateInputs then local controlbox=ents.FindByName('wiremain_b11c')[1];local door=ents.FindByName('b11_celldoor1')[1];controlbox.LightEdge=false;controlbox.Inputs=Wire_CreateInputs(controlbox,{'Door 1 - Control','Door 1 - Toggle'});function controlbox:TriggerInput(iname,value) if(iname=='Door 1 - Control') then if(value>0) then door:Fire('Open','',0) else door:Fire('Close','',0) end else if (value>0) and not self.LightEdge then door:Fire('Toggle','',0) end self.LightEdge=value>0 end end end

Reply:
Should be fine in a single line ig

Reply:
👍